packages/plugins/rubygem-foreman_azure_rm/rubygem-foreman_azure_rm.spec

```diff
@@ -111,11 +112,15 @@ cp -a .%{gem_dir}/* \
 %{gem_instdir}/Rakefile
 
 %posttrans
+%{foreman_db_migrate}
```

Comment:
@evgeni @zjhuntin Adding the rake task here for db migrate. Let me know if `db` needs to be mentioned at other places too.